Kajabi/sage-lib
# Labels in Dropdowns

## Description

Upcoming work on People list page and the Panel Controls' Bulk Actions requires adding heading items to dropdown lists.

### Screenshots


|  Spec  |  Browser  |
|--------|--------|
| <img width="365" alt="Screen Shot 2020-12-07 at 11 01 20 PM" src="https://user-images.githubusercontent.com/17955295/101438322-20db3a00-38e0-11eb-9f4d-4eec3c628746.png"> | <img width="312" alt="Screen Shot 2020-12-07 at 11 01 52 PM" src="https://user-images.githubusercontent.com/17955295/101438387-423c2600-38e0-11eb-873d-69d43216cc0f.png"> (Scrolls) |


### Steps for testing
1. See the Dropdown Object in the Documentation site and the new entry for Dropdown menu with Headings.
1. See the Dropdown component in Storybook and the new entry for Dropdown menu with Headings.
